SpaceWarpMod/UI/ConsoleWindow.cs

```diff
@@ -73,7 +97,18 @@ public class ConsoleWindow : ResizableWindow {
                 }
             }
 
-            GUI.Label(new Rect(15, 28, windowRect.width - 30, windowRect.height - 63), "", terminalImageStyle);
+            if (windowRect.width > 400) {
+                string label = Game.Input.asset.enabled ? "Disable Game Input" : "Enable Game Input";
```

Comment:
I am not sure how yet, but we really have to do something about this eventually ;)